samcli/lib/deploy/deployer.py

```diff
@@ -113,6 +113,7 @@ def has_stack(self, stack_name):
             if "Stack with id {0} does not exist".format(stack_name) in str(e):
                 LOG.debug("Stack with id %s does not exist", stack_name)
                 return False
+            return None
```

Comment:
Why are we returning `None` for a function which returns boolean? Can we return `True` or `False`?

Reply:
We did this so that this PR doesn't change any behavior. I believe the intention of the original author was to raise the exception at line 116 but we didn't confirm that

Reply:
So if you don't return anything from a function, it will return `None`?